app/src/main/java/org/mozilla/fenix/library/history/HistoryController.kt

```diff
@@ -54,6 +64,16 @@ interface HistoryController {
      * Navigates to [HistoryFragment] that would display history synced from other devices.
      */
     fun handleEnterSyncedHistory()
+
+    /**
+     * Navigates the user to [TurnOnSyncFragment].
+     */
+    fun handleSignIn()
+
+    /**
+     * Invokes [DefaultHistoryController.createAccount] that starts the authentication process.
```

Comment:
```
    /**
     * Handles a user's request to create an account.
     */
    fun handleCreateAccount()
```
I agree, but it makes me questioning if writing comments to every single public function makes sense, because this particular comment adds so little to the table.

Reply:
I agree with @MozillaNoah that the kDocs for the controller could be more generic and as they are right now it is probably just repeating what the implementation is doing verbatim, which will create some issues since interfaces are meant to be reusable and generic.

Given our interactor and controller architecture, it will feel at times that we are just repeating the same thing between the controller and interactor.  So, I can totally relate that the suggested comment will feel like it is adding very little value, but I think it also makes sense that the comment should be generic and not have to reference its descendants or any of the internal code.

My recommendation here is to make the interactors kDoc more meaningful and just reference the interactor kDoc in the controller. You could follow the kDoc style in `SessionControlController` for your controller here and use the existing convention with `@see [OnboardingInteractor.showOnboardingDialog]` to avoid repeating yourself between the interactor and controller. 

For kDoc for a function in an interactors, it should ideally describe what user interaction invokes the function call (eg, Invoked by an user click on "What's new" menu item) and what happens from the user's point of view (eg, Navigates to the what's new page). These functions are just event handler functions so it should just describe the events they are handling, and lightly describe what happens. I think it should be described from the user's point of view because now anyone can easily read the interface and understand what steps you need to execute to invoke the function described.